docker-compose.yml

```diff
@@ -1,10 +1,4 @@
 services:
-  postgresql:
-    image: postgres:12
-    ports:
-      - "5472:5432"
-    env_file:
-      - env.d/development/postgresql
 
   mysql:
```

Comment:
You don't have to remove this service as you are redifining it in `docker-compose-mysql.yml` ?

Reply:
Yes, I forgot to remove the mysql service on docker-compose.yml.